MonoGame.Framework/Audio/SoundEffectInstance.OpenAL.cs

```diff
@@ -260,14 +308,44 @@ private void PlatformSetVolume(float value)
 
         internal void PlatformSetReverbMix(float mix)
         {
+#if !MONOMAC
+            if (!OpenALSoundController.Efx.IsInitialized)
+                return;
+            reverb = mix;
+#endif
```

Comment:
Same as with `PlatformSetFilter`... this should immediately affect a sound that is currently playing.